build/crds/generate-crd-docs.sh

```diff
@@ -5,19 +5,21 @@ GEN_CRD_API_REFERENCE_DOC_VERSION=v0.3.0
 SCRIPT_CHECK_DOCS_DIFF=$(git diff --name-only --diff-filter=M | grep -Ec '/types\.go$')
 
 install_generator() {
-    go install github.com/ahmetb/gen-crd-api-reference-docs@${GEN_CRD_API_REFERENCE_DOC_VERSION}
+  go install github.com/ahmetb/gen-crd-api-reference-docs@${GEN_CRD_API_REFERENCE_DOC_VERSION}
 }
 
 run_gen() {
-  if [[  "$SCRIPT_CHECK_DOCS_DIFF" -gt 0 || "$1" = "--force" ]]; then
-    echo "differences found in types.go, rebuilding specification.md"
+  if [[ $SKIP_GEN_CRD_DOCS = true ]]; then
+    echo "SKIP_GEN_CRD_DOCS is set, skipping CRD docs generation"
+  elif [[ "$SCRIPT_CHECK_DOCS_DIFF" -gt 0 || "$1" = "--force" ]]; then
```

Comment:
Let's make this as simple as possible and have `make crds` always generate the docs. Let's not even check if types.go is checked out. Let's just always generate it.

```suggestion
  else
```

Reply:
Per discussion, we're going to remove that commit hash from the generated doc so we can avoid this issue

Reply:
@travisn updated with this PR, please check